blablas/ep7/index.md

```diff
@@ -3,7 +3,8 @@ date: 2018-10-25
 time: 20h
 duration: "1:22:38"
 title: "Métier de développeur"
-tags: ["dev"]
+tags: ["dev", "career"]
+category: "MSS"
```

Comment:
should be career

Reply:
@yjose my bad :laughing: . Fixed now :+1: